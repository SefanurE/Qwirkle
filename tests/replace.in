1
EWAN
MAX
replace p5
