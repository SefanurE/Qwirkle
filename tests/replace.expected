
Welcome to Qwirkle!
-------------------

Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit

> 
Starting a New Game

Enter a name for player 1 (uppercase letters only)
> 
Enter a name for player 2 (uppercase letters only)
> 
Let's Play!

EWAN, it's your turn
Score for EWAN: 0
Score for MAX: 0
   0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19
  -------------------------------------------------------------
A |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
B |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
C |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
D |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
E |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
F |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
G |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
H |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
I |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
J |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
K |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
L |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
M |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
N |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
O |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
P |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
Q |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
R |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
S |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
T |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |

Your hand is
P5, B5, G5, Y5, O5, R5

> 
EWAN, it's your turn
Score for EWAN: 0
Score for MAX: 0
   0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19
  -------------------------------------------------------------
A |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
B |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
C |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
D |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
E |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
F |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
G |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
H |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
I |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
J |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
K |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
L |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
M |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
N |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
O |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
P |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
Q |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
R |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
S |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
T |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |

Your hand is
P3, B5, G5, Y5, O5, R5

> Goodbye
